evm-bridging/main.go

```diff
@@ -229,13 +243,55 @@ func (p *provider) getContractBytecodeFromABIFile(contractName string) string {
 }
 
 func generateProxyEncodedConstructorData(implementationAddr, abiEncodedInitializeFunctionCall string) string {
+	implementationAddr = strings.TrimPrefix(implementationAddr, "0x")
+	abiEncodedInitializeFunctionCall = strings.TrimPrefix(abiEncodedInitializeFunctionCall, "0x")
+	fmt.Printf("abiEncodedInitializeFunctionCall is like this: %+v\n", abiEncodedInitializeFunctionCall)
+	//Run 'cast abi-encode'
+	initCallBytes, err := hex.DecodeString(abiEncodedInitializeFunctionCall)
+	if err != nil {
+		log.Fatalf("Failed to decode init call hex: %v", err)
+	}
+	const contractABI = `[{
+		"type": "constructor",
+		"inputs": [
+			{"name": "implementation", "type": "address"},
+			{"name": "initializeCall", "type": "bytes"}
```

Comment:
Doesn't seem to be making a difference but the second constructor param is called `_data`